tests/trainers/README.md

```diff
@@ -0,0 +1,2 @@
+# test-for-py
+tests for python
```

Comment:
You can omit this file, this really adds no additional information.

Reply:
I was going to update the file with some description of those new modules

Reply:
Ah, thanks, that would be much better of course.

Reply:
README file updated